### ISSUE TITLE
Save content items ajaxically

### ISSUE THREAD

**Issue by ryzr** [Noodlewitt/bootlegcms#27]
Refreshing the entire page can take time if you're deep within the content tree. Saving via ajax would be more seamless.

**Comment by Noodlewitt** [comment]
yes, this will be implemented with the upcoming language support - see 8e9cd9b33c451c6088c4c5b937409bef906bd9f0 for work in progress approach
